apps/dg/models/attribute_model.js

```diff
@@ -226,7 +226,7 @@ DG.Attribute = DG.BaseModel.extend(
         DG.assert(!SC.none( tCategories) && SC.isArray( tCategories.__order));
         tCategories.__order.forEach( function( iCategory, iIndex) {
           var tColor = tCategories[iCategory];
-          tColor = tColor.colorString || tColor;
+          tColor = tColor && tColor.colorString;
```

Comment:
It is frequently the case that tColor is already a string, in which case that's what we want to go with. It is not required that tColor have a property colorString, but if it does, that's what we want to go with.